chrysom/basicClient.go

```diff
@@ -111,13 +110,13 @@ type Items []model.Item
 // NewBasicClient creates a new BasicClient that can be used to
 // make requests to Argus.
 func NewBasicClient(config BasicClientConfig,
-	getLogger func(context.Context) log.Logger) (*BasicClient, error) {
+	getLogger sallust.GetLoggerFunc) (*BasicClient, error) {
```

Comment:
This type has been deprecated and removed.  Declaring it inline, as in the old code, is the way to go.